src/main/java/com/xceptance/xlt/engine/htmlunit/okhttp3/AuthenticatorImpl.java

```diff
@@ -65,27 +108,15 @@ public AuthenticatorImpl(final CredentialsProvider credentialsProvider)
         // check each provided challenge
         for (final Challenge challenge : response.challenges())
         {
-            // a special scheme that OkHttp uses for preemptive HTTPS proxy authentication
+            // preemptive basic authentication for HTTPS proxies using a special scheme
```

Comment:
The previous comment gave more insight IMO.

What about to change it as follows:
```suggestion
            // preemptive basic authentication for HTTPS proxies using a special scheme used by OkHttp only
```

Reply:
Done.